client/src/components/DatasetProjectCard.js

```diff
@@ -0,0 +1,165 @@
+import { Badge } from 'components/Badge'
+import { Button } from 'components/Button'
+import { Link } from 'components/Link'
+import { WarningText } from 'components/WarningText'
+import { getReadable } from 'helpers/getReadable'
+import { uniqueArray } from 'helpers/uniqueArray'
+
+// NOTE: Temporarily defined within this component (the existing project card's header component is in a separate file)
+const DatasetHeader = ({ dataset, linked = false }) => (
+  <>
+    {linked ? (
+      <Link href="#demo">
+        <Text weight="bold" color="brand" size="large">
+          {dataset.title}
+        </Text>
+      </Link>
+    ) : (
+      <Text weight="bold" color="brand" size="large">
+        {dataset.title}
+      </Text>
+    )}
+  </>
+)
+
+const Label = ({ label }) => <Text weight="bold">{label}</Text>
+
+// NOTE: This component accepts 'dataset' prop but it's subject to change
+// Currently mock data is used via Storybook for development
+export const DatasetProjectCard = ({ dataset }) => {
+  const { responsive } = useResponsive()
+  const projectIds = Object.keys(dataset.data)
+  const modalities = ['SINGLE_CELL', 'SPATIAL']
+  const downloadOptions = [
+    {
+      title: 'Data Format',
+      value: getReadable(dataset.format)
+    },
+    {
+      title: 'Modality',
+      value: modalities.map((modality) => {
+        const totalSamples = projectIds.reduce((acc, projectId) => {
+          return acc + dataset.data[projectId][modality].length
+        }, 0)
+
+        return (
+          <Box key={modality}>
+            <Text>
+              {totalSamples > 0 && `${getReadable(modality)} (${totalSamples})`}
+            </Text>
+          </Box>
+        )
+      })
+    },
+    {
+      title: 'Other Options',
+      value: projectIds.map((projectId) => {
+        const {
+          includes_bulk: includesBulk,
+          merge_single_cell: mergedSingleCell
+        } = dataset.data[projectId]
+        const hasNoOptions = !includesBulk && !mergedSingleCell
+
+        return (
+          <Box key={projectId}>
+            {includesBulk && (
+              <Text>Include all bulk RNA-seq data in the project</Text>
+            )}
+            {mergedSingleCell && (
+              <Text>Merge single-cell samples into 1 object</Text>
+            )}
+            {hasNoOptions && <Text>Not Specified</Text>}
+          </Box>
+        )
+      })
+    }
+  ]
+
+  const samplesDifferenceCount = projectIds.reduce((acc, projectId) => {
+    const { SINGLE_CELL, SPATIAL } = dataset.data[projectId]
+    const singleCellSamples = uniqueArray(SINGLE_CELL)
+    const spatialSamples = uniqueArray(SPATIAL)
+
+    if (singleCellSamples.length === 0 || spatialSamples.length === 0) {
+      return 0
+    }
+
+    const difference = [
+      ...singleCellSamples.filter((sample) => !spatialSamples.includes(sample)),
+      ...spatialSamples.filter((id) => !singleCellSamples.includes(id))
+    ]
+    return acc + difference.length
+  }, 0)
```

Comment:
This will be available on the `dataset.stats[projectId]`

Reply:
I've included `stats`  in the mock `datasets` response from the meeting note:

```js
"stats": {
        "projects": {
          // each of the project cards here
          "SCPCP000009": {
            "diagnoses_counts": "...",
            "samples_difference_count": 5 // Temporarily added here
          }
        }
```